flow-nodes/api-builder-plugin-petstore/test.js

```diff
@@ -0,0 +1,74 @@
+const { expect } = require('chai');
+const { MockRuntime } = require('@axway/api-builder-sdk');
+
+const getPlugin = require('./index');
+
+function getLogger () {
+	const logger = {
+		debug: () => {},
+		trace: () => {},
+		info: () => {},
+		warn: () => {},
+		error: () => {},
+		fatal: () => {}
+	};
+	logger.scope = () => logger;
+	return logger;
+}
+
+describe('flow-node format-date', () => {
+	let runtime;
+	before(async () => {
+		const plugin = await getPlugin({}, {
+			logger: getLogger()
+		});
+		runtime = new MockRuntime(plugin);
+	});
+
+	describe('#constructor', () => {
```

Comment:
The test suite fails with  Error: No @axway/api-builder-runtime instance which is peer dependency